contracts/contracts/strategies/balancer/BaseBalancerStrategy.sol

```diff
@@ -270,17 +275,15 @@ abstract contract BaseBalancerStrategy is InitializableAbstractStrategy {
         virtual
         returns (uint256 bptExpected)
     {
-        // Get the oracle from the OETH Vault
-        address priceProvider = IVault(vaultAddress).priceProvider();
+        require(
+            _assets.length == _amounts.length,
+            "Assets & amounts length mismatch"
```

Comment:
Its good to keep strings to 31 or less characters so they fit in an EVM word (32 bytes). The extra byte is for the string length.
`Assets & amounts mismatch` will works

Reply:
Oh right it is 31 not 32. Thanks corrected